src/CADL.Extension/Emitter.Csharp/src/index.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+export * from "./emitter.js";
```

Comment:
Could we change to TS type? It is strange for a ts file to have the knowledge of a js file.

Reply:
No. It is import/export of ES6 module type. What you mentioned without .js is commonjs. But cadl compile use ES6, we need to keep consistent otherwise it will casue runtime error: fail to load module.